mocking/file-logger/go/logger_test.go

```diff
@@ -0,0 +1,221 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+	"time"
+)
+
+type InMemoryFileStorage struct {
+	Logs             map[string][]string
+	ModificationTime map[string]time.Time
+}
+
+func (s *InMemoryFileStorage) AppendStringToFile(path, message string) error {
+	if s.Logs == nil {
```

Comment:
if you need those checks, then maybe better consider using constructor function:
```
func NewInMemoryFileStorage() *InMemoryFileStorage {
    return &InMemoryFileStorage{Logs: make....
}
```